templates/hound.yml

```diff
@@ -0,0 +1,11 @@
+# See https://houndci.com/configuration for configuration details.
+coffeescript:
+  enabled: true
+haml:
+  enabled: true
+javascript:
+  enabled: false
+ruby:
+  enabled: true
+scss:
+  enabled: true
```

Comment:
Yup, explicitness, easy to toggle on and off by editing the file rather than looking up the docs.